.ci/test.sh

```diff
@@ -53,16 +53,38 @@ fi
 if [[ $TASK == "lint" ]]; then
     conda install -q -y -n $CONDA_ENV \
         pycodestyle \
-        pydocstyle \
-        r-lintr
+        pydocstyle
     pip install --user cpplint
     echo "Linting Python code"
     pycodestyle --ignore=E501,W503 --exclude=./compute,./.nuget . || exit -1
     pydocstyle --convention=numpy --add-ignore=D105 --match-dir="^(?!^compute|test|example).*" --match="(?!^test_|setup).*\.py" . || exit -1
-    echo "Linting R code"
+
+    # set up R environment
+    # echo "Linting R code"
+    # R_LIB_PATH=~/Rlib
+    # mkdir -p $R_LIB_PATH
+    # echo "R_LIBS=$R_LIB_PATH" > ${HOME}/.Renviron
+    # echo 'options(repos = "https://cran.rstudio.com")' > ${HOME}/.Rprofile
+    # export PATH="$R_LIB_PATH/R/bin:$PATH"
+    # # installing precompiled R for Ubuntu
+    # # https://cran.r-project.org/bin/linux/ubuntu/#installation
+    # sudo apt-get update
+    # sudo add-apt-repository \
+    #     "deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/"
+    # sudo apt-get update
+    # sudo apt-get install \
+    #     -y \
+    #         r-base \
+    #         r-base-dev \
+    #         || exit -1
```

Comment:
I am testing `lintr` install from CRAN. If that works, will remove the commented code here

Reply:
it worked!